Makefile

```diff
@@ -12,25 +39,27 @@ clean:
 
 .PHONY: images
 images:
-	@docker build -f Dockerfiles/centos6 -t $(IMAGE)/centos6 .
-	@docker build -f Dockerfiles/centos7 -t $(IMAGE)/centos7 .
-	@docker build -f Dockerfiles/centos8 -t $(IMAGE)/centos8 .
+	@docker build -f Dockerfiles/centos6.Dockerfile -t $(IMAGE)/centos6 .
+	@docker build -f Dockerfiles/centos7.Dockerfile -t $(IMAGE)/centos7 .
+	@docker build -f Dockerfiles/centos8.Dockerfile -t $(IMAGE)/centos8 .
```

Comment:
Building the centos 8 image fails on:
```
  Upgrading        : filesystem-3.8-3.el8.x86_64                         13/256 
Error unpacking rpm package filesystem-3.8-3.el8.x86_64
  Installing       : glibc-langpack-en-2.28-127.el8.x86_64               14/256 
error: unpacking of archive failed on file /proc: cpio: chown
error: filesystem-3.8-3.el8.x86_64: install failed
```
It's similar to other issues we've seen with installing filesystem: https://issues.redhat.com/browse/OAMG-4310
Is the `/proc/` read-writeable?

Reply:
The docker image is being updated from CentOS 8.2 to 8.3:
```
STEP 10: RUN dnf update -y && dnf install -y $APP_MAIN_DEPS && dnf clean all
CentOS-8 - AppStream                            952 kB/s | 6.3 MB     00:06    
CentOS-8 - Base                                 926 kB/s | 2.3 MB     00:02    
CentOS-8 - Extras                                31 kB/s | 8.6 kB     00:00    
Dependencies resolved.
================================================================================
 Package                     Arch   Version                     Repo       Size
================================================================================
Installing:
 centos-linux-release        noarch 8.3-1.2011.el8              BaseOS     22 k
     replacing  centos-release.x86_64 8.2-2.2004.0.1.el8
     replacing  centos-repos.x86_64 8.2-2.2004.0.1.el8
```
Is it because my previous image was CentOS 8.2? How can I remove it completely so it does not try to get updated to 8.3? 
Per https://hub.docker.com/_/centos the latest image available should be 8.3.

Reply:
I believe it used a cached version of the base image (centos:8) and then tries to update it. I think my proposal of building with --no-cache option should be enough to resolve your issue